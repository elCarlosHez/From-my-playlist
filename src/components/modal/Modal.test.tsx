import { create } from "react-test-renderer";
import { Modal } from "./";

describe('<Modal /> Tests', () => {
  it("Render the Header component.", () => {
    const component = create(
      <Modal onClose={() => {}}>
        <p>Hello Testers!</p>
      </Modal>
    );
    const json = component.toJSON();
    expect(json).toBeDefined();
    expect(json).toMatchSnapshot();
  });
});

