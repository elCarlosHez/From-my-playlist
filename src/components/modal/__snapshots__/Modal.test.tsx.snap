// Vitest Snapshot v1

exports[`<Modal /> Tests > Render the Header component. 1`] = `
<aside
  className="absolute top-0 left-0 w-screen h-screen z-10 bg-slate-100 px-5"
>
  <button
    className="absolute top-10 right-10 z-20"
    onClick={[Function]}
    type="button"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-xmark fa-2x "
      data-icon="xmark"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={{}}
      viewBox="0 0 320 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M310.6 361.4c12.5 12.5 12.5 32.75 0 45.25C304.4 412.9 296.2 416 288 416s-16.38-3.125-22.62-9.375L160 301.3L54.63 406.6C48.38 412.9 40.19 416 32 416S15.63 412.9 9.375 406.6c-12.5-12.5-12.5-32.75 0-45.25l105.4-105.4L9.375 150.6c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L160 210.8l105.4-105.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-105.4 105.4L310.6 361.4z"
        fill="currentColor"
        style={{}}
      />
    </svg>
  </button>
  <p>
    Hello Testers!
  </p>
</aside>
`;
